juniper_official/routing/check-isis-adjacency-status.rule

```diff
@@ -111,66 +134,53 @@
                     juniper {
                         operating-system junos {
-                            products ACX {
-                                platforms All {
-                                    releases 22.1R1 {
-                                        release-support min-supported-release;
-                                    }
-                                }
-                            }
-                            products EX {
-                                platforms All {
-                                    releases 15.1R1 {
-                                        release-support min-supported-release;
-                                    }
-                                }
-                            }
                             products MX {
-                                platforms All {
-                                    releases 15.1R1 {
+                                sensors isis-oc-junos;
+                                platforms MX304 {
+                                    releases 23.2R1 {
                                         release-support min-supported-release;
                                     }
                                 }
-                            }
-                            products PTX {
-                                platforms All {
-                                    releases 15.1R1 {
+                                platforms MX204 {
+                                    releases 23.2R1 {
                                         release-support min-supported-release;
                                     }
-                                }
-                            }
-                            products QFX {
-                                platforms All {
-                                    releases 15.1R1 {
-                                        release-support min-supported-release;
-                                    }
-                                }
-                            }
-                            products ACX {
-                                platforms All {
-                                    releases 22.1R1 {
-                                        release-support min-supported-release;
-                                    }
-                                }
+                                }								
                             }
                         }
                         operating-system junosEvolved {
                             products ACX {
-                                platforms All {
+                                sensors isis-oc;							
+                                platforms ACX7024 {
                                     releases 22.3R1 {
+                                        sensors isis-oc;									
                                         release-support min-supported-release;
                                     }
+                                    releases 23.2R1 {
```

Comment:
shouldn't we add this release first? Did you test this on DV. Can you test and let me know?

Reply:
Modified and tested.